deepmd/dpmodel/output_def.py

```diff
@@ -261,9 +295,15 @@ def do_reduce(
     def_redu: Dict[str, OutputVariableDef] = {}
     for kk, vv in def_outp_data.items():
         if vv.reduciable:
+            assert vv.category & OutputVariableOperation.REDU.value == 0
             rk = get_reduce_name(kk)
             def_redu[rk] = OutputVariableDef(
-                rk, vv.shape, reduciable=False, differentiable=False, atomic=False
+                rk,
+                vv.shape,
+                reduciable=False,
+                differentiable=False,
+                atomic=False,
+                category=vv.category | OutputVariableOperation.REDU.value,
```

Comment:
Torchscript is a shit! I have to use `OutputVariableOperation.REDU.value` instead of `OutputVariableOperation.REDU` to let the jit compile.

Reply:
introduce something like 
`apply_operation(vv, OutputVariableOperation.XXX)` and `check_operation_applied(vv, OutputVariableOperation.XXX)`  to solve the issue?